services/src/main/java/edu/unc/lib/dl/cdr/services/rest/modify/MoveObjectsController.java

```diff
@@ -85,8 +127,12 @@ public void setDigitalObjectManager(DigitalObjectManager digitalObjectManager) {
 	public static class MoveRequest {
 		private String newParent;
```

Comment:
I would prefer that too, it just that it was already using the name newParent before this PR.  Changing the names of fields in this class will require updating things on the javascript side as well, which is fine, I was just avoiding extra changes.  I'll revisit the names though.